tokio/src/runtime/blocking/mod.rs

```diff
@@ -262,7 +262,7 @@ impl Drop for PoolWaiter {
 ///
 /// ```
 /// # async fn docs() {
-/// tokio::executor::blocking::in_place(move || {
+/// tokio::runtime::blocking::in_place(move || {
```

Comment:
WDYT about reexporting `blocking` at the top level, so this could be `tokio::blocking::in_place`?

Reply:
I guess it requires the `runtime` feature flag, so this makes sense. I just thought a less deeply nested path could improve ergonomics...:woman_shrugging:

Reply:
I think it would make sense to provide it under just `tokio::blocking` and under the runtime flag. We should probably use those nice doc_cfg's anyways so it should be easy to discover.

Reply:
Yep, sounds good. Follow up PR though!